server/routes/org_routes.py

```
@@ -382,6 +382,16 @@ def add_member(user_id):
     if role not in VALID_ROLES:
         return jsonify({"error": "Invalid role"}), 400
 
+    # Hook: check if org can add more members (seat limit enforcement)
+    from utils.hooks import get_hook
+    with db_pool.get_admin_connection() as conn:
+        with conn.cursor() as cur:
```

Ruff (0.15.15)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)